### PR TITLE
Validation schema for config policies' decisions

### PR DESCRIPTION
**Pull request by shlomitk1**
Closes https://github.com/fybrik/fybrik/issues/1199

This PR adds a validation schema for admin policies' decisions and validates them upon evaluation.
Also, a policy structure has been changed: 
- {"capability": capability, "decision": decision} instead of {capability: decision}
- deployment status: False, True, Unknown (not to confuse with boolean true and false)

Note: I've added a manual DeepCopy functionality because of an issue with auto-generated code that does not pass gosec checks, will open an issue soon.

### CODE REVIEW

File: charts/fybrik/files/adminconfig/default_policies.rego [hunk 1]
```diff
@@ -3,22 +3,38 @@ package adminconfig
...
-    policy := {"ID": "read-default", "description":"Read capability is requested for read workloads", "version": "0.1"}
-    decision := {"policy": policy, "deploy": read_request}
+    policy := {"ID": "read-default-enabled", "description":"Read capability is requested for read workloads", "version": "0.1"}
+    decision := {"policy": policy, "deploy": "True"}
+}
+
+# read capability deployment
+config[{"capability": "read", "decision": decision}] {
+    input.request.usage.read == false
+    read_request := input.request.usage.read
```
Comment (revit13) [inline charts/fybrik/files/adminconfig/default_policies.rego:16]: read_request does not seem to be used.

File: charts/fybrik/files/adminconfig/default_policies.rego [hunk 2]
```diff
@@ -3,22 +3,38 @@ package adminconfig
 # out-of-the-box policies
 
 # read capability deployment
-config[{"read": decision}] {
+config[{"capability": "read", "decision": decision}] {
+    input.request.usage.read == true
     read_request := input.request.usage.read
-    policy := {"ID": "read-default", "description":"Read capability is requested for read workloads", "version": "0.1"}
```
Comment (revit13) [inline charts/fybrik/files/adminconfig/default_policies.rego:8]: read_request does not seem to be used.

File: charts/fybrik/files/adminconfig/default_policies.rego (hunk 1 above)
Comment (shlomitk1) [reply charts/fybrik/files/adminconfig/default_policies.rego]: Thanks, removed.

File: charts/fybrik/files/adminconfig/default_policies.rego (hunk 2 above)
Comment (shlomitk1) [reply charts/fybrik/files/adminconfig/default_policies.rego:8]: Removed

**Review by revit13 — APPROVED** [review]

**Review by simanadler — APPROVED** [review]

**Pull request merged**